definitions/core/es5-1/README.md

```diff
@@ -0,0 +1,9 @@
+An alternative JS core library definition for ES5.1
+
+The Flow core library definitions cover the latest ES versions. That has a lot of additional methods and classes compared to ES5 and so code-completion and type checking do not give errors about using those new methods and classes. Use this core library definition to write pure ES5.1 Javascript without having to add pollyfills or compile down to ES5 via Babel.
+
+Requires the `no_flowlib` option.
```

Comment:
I'm not sure this would be clear to someone unfamiliar with Flow, maybe clarify which option this is referring to, or link to the docs?

Reply:
Actually we should ignore everything inside `core/es5-1` that just copied from an existing definition that was in `definition/npm` but isn't actually a npm package